apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/familieforhold/SamsvarPersongalleri.tsx

```diff
@@ -47,36 +50,82 @@ const PersonUtenIdentWrapper = styled.div`
   const { samsvar, saktype } = props
+  const visMismatchPdl = useFeatureEnabledMedDefault('familieforhold-vis-mismatch-pdl', false)
+
   const personerUtenIdenterSak = samsvar.persongalleri?.personerUtenIdent ?? []
   const personerUtenIdenterPdl = samsvar.persongalleriPdl?.personerUtenIdent ?? []
-  const harPersonerUtenIdenter = personerUtenIdenterPdl.length > 0 || personerUtenIdenterSak.length > 0
 
-  if (!harPersonerUtenIdenter) {
+  const harPersonerUtenIdenter = samsvar.problemer.includes('HAR_PERSONER_UTEN_IDENTER')
+  const harAvvikMotPdl =
+    samsvar.problemer.filter(
+      (problem) =>
+        problem !== 'HAR_PERSONER_UTEN_IDENTER' && problem !== 'EKSTRA_SOESKEN' && problem !== 'MANGLER_SOESKEN'
+    ).length > 0
+
+  if (samsvar.problemer.length === 0) {
     return null
   }
 
   return (
-    <div>
-      <Alert variant="warning">
-        Det er personer uten identer i PDL i saksgrunnlaget. Disse personene kan ikke brukes i beregning av trygdetid
-        eller en eventuell søskenjustering på gammelt regelverk.
-      </Alert>
-      <Heading size="small" level="4">
-        Personer uten identer i saksgrunnlag:
-      </Heading>
-      <PersonerUtenIdenterVisning saktype={saktype} personer={personerUtenIdenterSak} />
-
-      <Heading size="small" level="4">
-        Personer uten identer i PDL:
-      </Heading>
-      <PersonerUtenIdenterVisning saktype={saktype} personer={personerUtenIdenterPdl} />
-    </div>
+    <>
+      {harAvvikMotPdl && visMismatchPdl && (
+        <MediumAdvarsel>
+          <BodyShort spacing>
+            Det er forskjeller mellom familieforholdet i behandlingen og det familieforholdet vi utleder ut i fra PDL.
+            Se nøye over og eventuelt korriger persongalleriet ved å redigere.
+          </BodyShort>
+          <Heading level="4" size="xsmall" spacing>
+            Familieforholdet i behandlingen
+          </Heading>
+          <FlexRow $spacing>
+            <Info label="Avdøde" tekst={samsvar.persongalleri.avdoed?.join(', ')} />
+            <Info label="Gjenlevende" tekst={samsvar.persongalleri.gjenlevende?.join(', ')} />
+            <Info
+              label="Kilde"
+              tekst={`${samsvar.kilde?.type.toUpperCase()} (${formaterKanskjeStringDato(samsvar.kilde?.tidspunkt)})`}
+            />
+          </FlexRow>
+
+          <Heading level="4" size="xsmall" spacing>
+            Familieforholdet i PDL
+          </Heading>
+          <FlexRow $spacing>
+            <Info label="Avdøde" tekst={samsvar.persongalleriPdl?.avdoed?.join(', ')} />
+            <Info label="Gjenlevende" tekst={samsvar.persongalleriPdl?.gjenlevende?.join(', ')} />
+            <Info
+              label="Kilde"
+              tekst={`${samsvar.kildePdl?.type.toUpperCase()} (${formaterKanskjeStringDato(samsvar.kildePdl?.tidspunkt)})`}
+            />
+          </FlexRow>
+        </MediumAdvarsel>
+      )}
+      {!harPersonerUtenIdenter && (
```

Comment:
Er det riktig med `!` her?
Teksten under ser ut til å henvise til at det finnes noen, eller leser jeg det feil?

Reply:
Ah, nice catch! var invertert for å teste 🤦

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/4194/commits/6682d7afb18a437444af559a0e1cf7450be28ccf